src/app/_components/UserListClient.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
+import { trpc } from '../_trpc/client';
+import { serverClient } from '../_trpc/serverClient';
+
+export default function UserListClient({
+  initialUsers
+}: {
+  initialUsers: Awaited<ReturnType<(typeof serverClient)['user']['list']>>;
```

Comment:
No need to actually make this change now; but be on the lookout for if you create types like this frequently. If so, consider creating some kind of util folder that has definitions for types you retrieve from the router in this fashion.